runtime/strategies/init-population.js

```diff
@@ -8,10 +8,15 @@
 import {Strategy} from '../../strategizer/strategizer.js';
 
 export default class InitPopulation extends Strategy {
-  constructor(arc) {
+  // Produces affordance-conforming recipes from the given arc's context, unless
+  // an optional list of recipes is specified in which case solely the recipes
+  // in that list are considered.
+  constructor(arc, optionalRecipes) {
```

Comment:
`optionalRecipes` is a confusing name, because they are actually higher priority. maybe always pass the recipes? or explicitly define all parameters - `recipes, slotComposer, loadedParticles`?